src/main/java/org/cascadebot/cascadebot/data/managers/CascadeUserDataManager.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2019 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package org.cascadebot.cascadebot.data.managers;
+
+import com.github.benmanes.caffeine.cache.Caffeine;
+import com.github.benmanes.caffeine.cache.LoadingCache;
+import org.bson.conversions.Bson;
+import org.cascadebot.cascadebot.CascadeBot;
+import org.cascadebot.cascadebot.data.database.DebugLogCallback;
+import org.cascadebot.cascadebot.data.objects.user.CascadeUser;
+
+import java.util.concurrent.TimeUnit;
+
+import static com.mongodb.client.model.Filters.eq;
+
+public class CascadeUserDataManager {
+
+    private static final String COLLECTION = "bot_users";
+    
+    private static LoadingCache<Long, CascadeUser> users = Caffeine.newBuilder()
```

Comment:
Make sure you add this cache to the Metrics

Reply:
the other cache isn't in the metrics?

Reply:
https://github.com/CascadeBot/CascadeBot/blob/dev/src/main/java/org/cascadebot/cascadebot/CascadeBot.java#L220